### DISCUSSION TITLE
Remove dependency on jquery

### DISCUSSION THREAD

**Discussion by Calabashmc** [makeusabrew/bootbox#849] [Ideas]
Now that Boostrap 5 is supported will the jquery dependency be removed?
Bootstrap and just about everyone else is moving away from jquery to vanilla js. I currenlty use sweetalert 2 for this reason but bootboxjs looks much better except for the jquery requirement.

**Comment by tiesont** [comment]
Probably. Not anytime soon, I'm afraid, but it's on my list.

**Reply by Calabashmc** [reply]
Excellent. I'll keep an eye on the project.

**Comment by w8tcha** [comment]
> Now that Boostrap 5 is supported will the jquery dependency be removed? Bootstrap and just about everyone else is moving away from jquery to vanilla js. I currenlty use sweetalert 2 for this reason but bootboxjs looks much better except for the jquery requirement.

Take a look at my fork, i completely removed jquery 

https://github.com/w8tcha/bootbox
